src/packages/@ncigdc/modern_components/FilesTable/FilesTable.model.js

```diff
@@ -201,7 +201,7 @@ const filesTableModel = [
   },
   {
 	  name: 'Participant ID',
-	  id: 'participant_id',
+	  id: 'cases.samples.participant_id',
```

Comment:
It deeply pains me that this was such a simple change given the time spent on it, bleh